### ISSUE TITLE
StructureMigration silently ignores rollback for testing env.

### ISSUE THREAD

**Issue by jlsjonas** [laravel-enso/StructureManager#7]
This is  a **bug**.

### Prerequisites
* [x] Are you running the latest version?
* [x] Are you reporting to the correct repository?
* [x] Did you check the documentation?
* [x] Did you perform a cursory search?

### Description

It looks like running a rollback on one of the Structuremigrations doesn't remove the items from the database anymore (possibly related to the changes made to fix #4 )

### Steps to Reproduce

1. Generate structure
2. Migrate
3. Rollback, note that the down migration executed successfully according to Laravel, but nothing happened.

### Expected behavior

Created structure to be removed (permissions, permission group, menuItem)

### Actual behavior

Nothing; except for the migration supposedly succeeding.

**Comment by aocneanu** [comment]
I can’t be related to the previous fix... I’ll look into it tomorrow.

**Comment by aocneanu** [comment]
I tried it three times, it worked as expected.

**Comment by jlsjonas** [comment]
Something Windows-specific maybe?

**Comment by aocneanu** [comment]
Shouldn't be related to the OS.

The structure migration class is pretty straighforward, you could try to `dd` it :) It would take a few minutes

[this](https://github.com/laravel-enso/StructureManager/blob/master/src/app/Classes/Destroyer.php) where you have to look and check if all the parameters are ok.

**Comment by jlsjonas** [comment]
Found the culprit... (and then I got interrupted by a fire drill...)
```php
public function down()
    {
        if (config('app.env') == 'testing') {
            return;
        }
```
should echo some kind of warning imho

**Comment by aocneanu** [comment]
The example.env is used for testing in our case, which is not the cleanest way to do it. I will add a `.testing.env` and leave the `.testing.env` unaltered.

**Comment by aocneanu** [comment]
closed with f60b252b9eae9db1c09733552c6509a8567e9b4c

**Comment by jlsjonas** [comment]
👍 (I'd default to `local` for `.env.example`, as this environment gives the richest _playing_ environment)
p.s.: for the testing environment you likely want to add `DEBUGBAR_ENABLED=false` to save some cycles (unless you use the debugbar's info for testing somehow?)
